### PR TITLE
LPS-169841 portal-search-web: Pass in current scope when fetching suggestions

### PR DESCRIPTION
**Pull request by oliv-yu**
https://issues.liferay.com/browse/LPS-169841 - "Let the User Choose" search bar scope is not honored with the search bar suggestions

Keeps the scope inside the fetch call up to date by directly passing it in.

### CODE REVIEW

**Comment by oliv-yu** [comment]
These test failures don't seem related, but let me know if you think they are? Thanks!

**Review by thektan — CHANGES_REQUESTED** [review]
Also I believe this is a typo if you could change this as well.

https://github.com/liferay-search/liferay-portal/pull/694/files#diff-9a251bfad674e84ebf27c67d0afd76a68233753cd93fe47bc929c080d1ba6a0cR129-R131

// Immediately show loading spinner unless the value hasn't changed.
// If the value hasn't changed, no new request will be made and the
// loading spinner will ~never be hidden~ not be shown.

File: modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/components/SearchBar.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ export default function SearchBar({
 	const alignElementRef = useRef();
 	const dropdownRef = useRef();
 
-	const _fetchSuggestions = (value) => {
+	const _fetchSuggestions = (value, currentScope) => {
```
[the remark this answers is not in the document]
Comment (oliv-yu) [reply modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/components/SearchBar.js]: That makes sense, added!

**Review by thektan — APPROVED** [review]
Thanks! Looks good!

**Comment by thektan** [comment]
ci:test:sf

**Comment by thektan** [comment]
ci:test:search

**Comment by thektan** [comment]
ci:test:relevant

**Comment by joshchong** [comment]
Sent to BChan:
https://github.com/brianchandotcom/liferay-portal/pull/127205

**Pull request closed without merging**